src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ValidationServiceFacadeImpl.java

```diff
@@ -159,35 +176,40 @@ public ExceptionCaseWorker createException(final long jobId, final String messag
     private CaseWorkerAudit createOrUpdateCaseworkerAudit(AuditStatus auditStatus, String fileName) {
         if (isNull(caseWorkerAudit) || isNull(caseWorkerAudit.getJobId())) {
             UserInfo userInfo = jwtGrantedAuthoritiesConverter.getUserInfo();
-            String userName = (nonNull(userInfo) && nonNull(userInfo.getName())) ? userInfo.getName() : EMPTY;
+            String userId = (nonNull(userInfo) && nonNull(userInfo.getUid())) ? userInfo.getUid() : EMPTY;
             caseWorkerAudit = CaseWorkerAudit.builder()
                 .status(auditStatus.getStatus())
-                .jobStartTime(LocalDateTime.now())
+                .jobStartTime(LocalDateTime.now(ZoneOffset.UTC))
                 .fileName(fileName)
```

Comment:
When persisting an OffsetDateTime entity attribute, Hibernate does not store the offset separately, hence the local offset is used. For this reason, the OffsetDateTime is not very useful, and it can be replaced by a LocalDateTime instead.(https://vladmihalcea.com/date-timestamp-jpa-hibernate/) Using Zonset here hence time stored in UTC